src/main/java/me/xmrvizzy/skyblocker/utils/render/title/TitleContainerConfigScreen.java

```diff
@@ -158,7 +159,7 @@ public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
     public void close() {
         SkyblockerConfig.get().general.titleContainer.x = (int) hudX;
         SkyblockerConfig.get().general.titleContainer.y = (int) hudY;
-        AutoConfig.getConfigHolder(SkyblockerConfig.class).save();
+        SkyblockerConfig.save();
```

Comment:
Do we need to make this go back to the parent screen too?

Reply:
I forgot about this one, I'll add the button for it and that functionality